engines/cge2/cge2.cpp

```diff
@@ -184,8 +182,9 @@ bool CGE2Engine::hasFeature(EngineFeature f) const {
 }
 
 Common::Error CGE2Engine::run() {
-    if (_ttsMan != nullptr)
-		_ttsMan->setLanguage(Common::getLanguageCode(getLanguage()));
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+    if (ttsMan != nullptr)
+		ttsMan->setLanguage(Common::getLanguageCode(getLanguage()));
```

Comment:
Indentation seems incorrect. Did you maybe use spaces instead of tabs on some of these lines? Line 186 seems to be the incorrect one.

Reply:
Fixed. I would really like to know what I did to make sporadic spaces appear in my code.....